### PR TITLE
feat(sr): add lvmohba SR type management

### PR DESCRIPTION
**Pull request by olivierlambert**
Related to https://github.com/vatesfr/xo-web/issues/1992

### CODE REVIEW

File: src/api/sr.js [hunk 1]
```diff
@@ -223,6 +223,51 @@ createNfs.resolve = {
...
+    nameDescription,
+    'lvmoohba', // SR LVM over HBA
+    'user', // recommended by Citrix
+    true,
+    {}
+  )
+
+  const sr = await xapi.call('SR.get_record', srRef)
+  return sr.uuid
+}
```
Comment (julien-f) [inline src/api/sr.js]: What's the permission on this methods?

Comment (olivierlambert) [inline src/api/sr.js]: Will do in another PR
